php/commands/super-admin.php

```diff
@@ -27,9 +27,10 @@ public function _list( $_, $assoc_args ) {
 	 * <user>...
 	 * : One or more user IDs, user emails, or user logins.
 	 */
-	public function add( $args, $_ ) {
+	public function add() {
+		$request = \WP_CLI\Request::get_from_argv();
 
-		$users = $this->fetcher->get_many( $args );
+		$users = $this->fetcher->get_many( $request->get_positional_args() );
```

Comment:
When called within the scope of the command, `get_positional_args()` is expected to be the positional arguments passed to the command, and not the command itself (e.g. `super-admin add`)